indexer/index.hpp

```diff
@@ -64,14 +68,16 @@ class Index : public MwmSet
 protected:
   /// MwmSet overrides:
   //@{
-  unique_ptr<MwmInfo> CreateInfo(platform::LocalCountryFile const & localFile) const override;
+  std::unique_ptr<MwmInfo> CreateInfo(platform::LocalCountryFile const & localFile) const override;
 
-  unique_ptr<MwmValueBase> CreateValue(MwmInfo & info) const override;
+  std::unique_ptr<MwmValueBase> CreateValue(MwmInfo & info) const override;
   //@}
 
 public:
+  std::unique_ptr<FeatureType> GetOriginalFeature(FeatureID const & id) const;
```

Comment:
Not sure that we need this in Index, in general, Index is responsible for mwm registering and handling, not in reading them.

Reply:
there is an idea about moving this into Index::FeaturesLoaderGuard. Will it be better?

Reply:
Yes, FeaturesLoaderGuard is generally used for loading features.